package.json

```diff
@@ -379,6 +379,7 @@
     "react-window": "^1.8.5",
     "redux-act": "^1.8.0",
     "redux-logger": "^3.0.6",
+    "redux-persist": "6.0.0",
```

Comment:
are there any alternative libraries that would work here which are more actively maintained?